src/package.js

```diff
@@ -0,0 +1,13 @@
+const loadMainPackageJSON = (attempts = 1) => {
+  if (attempts > 5) {
+    throw new Error('Can\'t resolve main package.json file');
+  }
+  const mainPath = attempts === 1 ? './' : Array(attempts).join('../');
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this usage of the ternary operator to an "if"/"else" structure. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AConditionalOperator)